.github/workflows/sync-rest.yml

```diff
@@ -0,0 +1,272 @@
+name: Sync Rest
+
+on:
+  workflow_dispatch:
+    inputs:
+      tracks:
+        description: 'List of tracks to rerun the syncing workflow for, formatted as JSON array, e.g. ["exercism/julia"]'
+        required: true
+        default: ''
+  repository_dispatch:
+    types: [track_changes]
```

Comment:
Do we want to have separate dispatch events now?

Reply:
It's not necessary, `sync-rest` works for all repos, including tracks, if triggered via dispatch events.